releasenotes/notes/apm-max-tps-0-value-0c5b09dfb820c6dd.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    APM Breaking change: The `default head based sampling mechanism <https://docs.datadoghq.com/tracing/trace_ingestion/mechanisms?tab=environmentvariables#head-based-default-mechanism>`_
+    settings `apm_config.max_traces_per_second` or `DD_APM_MAX_TPS` when set to 0 will result in 
+    rates of 0% instead of 100% on head based sampling. 
```

Comment:
```suggestion
    settings `apm_config.max_traces_per_second` or `DD_APM_MAX_TPS`, when set to 0, will be sending 
    0% of traces to Datadog, instead of 100% in previous agent versions. 
```